db/migrate/20210707143210_create_inventories.rb

```diff
@@ -0,0 +1,19 @@
+class CreateInventories < ActiveRecord::Migration[6.1]
+  def change
+    create_table :inventories do |t|
+      t.integer :memory_size, null: false, unsigned: true
+      t.integer :manufactoring_year, null: false, unsigned: true
+      t.string :os_version, null: false
+      t.string :color, null: false
+      t.decimal :price, null: false, unsigned: true
+      t.decimal :original_price, unsigned: true
+      t.integer :status, default: 0   # Active
```

Comment:
IMO, status filed should add the validation `null: false`

Reply:
Done!